pkg/reconciler/proxy/proxy.go

```diff
@@ -402,16 +402,23 @@ func updateVolume(pod corev1.Pod, volumeName, configmapName, key string) corev1.
 			}
 		}
 
-		// /etc/ssl/certs is the default place where CA certs reside in *nix
-		// however this can be overridden using SSL_CERT_DIR, let's check for
-		// that here.
-		sslCertDir := "/etc/ssl/certs"
+		// we wiil mount the certs at this location so we don't override the existing certs
+		sslCertDir := "/custom-certs"
+		envAvaiable := false
```

Comment:
nit:
```suggestion
		certEnvAvailable := false
```
for readability

Reply:
resolved